researchapp/services/logging.py

```diff
@@ -0,0 +1,42 @@
+""" Log stuff to ElasticSearch so we can look at it later. """
+import datetime
+import json
+import requests
+
+
+ES_URL = 'https://search-s4s-logs-xsjsafiwd7vkpiucmjqmdjkp7y.us-west-2.es.amazonaws.com/' \
```

Comment:
Hmm. Can you add it to a non-distributed config file if this is really necessary? It would be better not to "leak" our own reference URLs to everybody who clone and tries to run the repo (I don't mean for security reasons, but just because a random clone of the repo shouldn't logically be trying to write to our logging server).